locales

```diff
@@ -2,3 +2,4 @@ en_US.UTF-8 UTF-8
 de_DE.UTF-8 UTF-8
 nl_NL.UTF-8 UTF-8
 fr_FR.UTF-8 UTF-8
+ro_RO.utf-8 UTF-8
```

Comment:
Could you capitalize the encoding pls like it's done for others(`.UTF-8`)? 

Feel free to merge after it.